cardano-cli/src/Cardano/CLI/EraBased/Commands/Genesis.hs

```diff
@@ -87,7 +87,7 @@ data GenesisCreateTestNetDataCmdArgs = GenesisCreateTestNetDataCmdArgs
   { specShelley :: !(Maybe FilePath) -- ^ Path to the @genesis-shelley@ file to use. If unspecified, a default one will be used if omitted.
   , numGenesisKeys :: !Word -- ^ The number of genesis keys credentials to create and write to disk.
   , numPools :: !Word -- ^ The number of stake pools credentials to create and write to disk.
-  , numStakeDelegators :: !Word -- ^ The number of delegators to pools to create and write to disk.
+  , stakeDelegators :: !(Bool, Word) -- ^ The number of delegators to pools to create. The Boolean indicates whether the credentials should be written to disk (True) or be transient (False)
```

Comment:
I'm not a fan of `Bool` due to boolean blindness. Introduce a sum type for this.

Reply:
Ah I see this is outdated.